Description: add debian/ files to MANIFEST to ensure tests pass
Author: Ryan Niebur <ryan@debian.org>
Origin: vendor
Forwarded: no
--- a/MANIFEST
+++ b/MANIFEST
@@ -11,3 +11,19 @@
 lib/Test/CheckManifest.pm
 
 META.yml
+debian/changelog
+debian/rules
+debian/watch
+debian/libtest-checkmanifest-perl.debhelper.log
+debian/copyright
+debian/compat
+debian/control
+debian/patches/series
+debian/patches/fix-tests
+debian/README.source
+.pc/.version
+.pc/applied-patches
+.pc/fix-tests/.timestamp
+.pc/fix-tests/MANIFEST
+.pc/.quilt_patches
+.pc/.quilt_series
